content/blog/2021-05-15-may-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,239 @@
+﻿---
+title: May '21 Heartbeat
+date: 2021-05-15
+description: |
+  Monthly updates are here! We've hit 30 team members! MLOps learning 
+  opportunities, tutorials with integrations, conference videos, Discord 
+  server growth, and more!
+descriptionLong: |
+  This month you will find:
+  🚀 info on our growing team 
+  📖 MLOps learning opportunities,
+  🔃 tutorials with integrations, 
+  🎥 conference videos from our team members, 
+  💥 Discord server growth, and 
+  more!
+picture: 2021-05-15/may21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/dvc-may-2021-heartbeat/
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - MLOps
+  - ML Repa
+  - Andrew Ng
+  - DAGsHub
+  - Curvenote
+---
+
+# DVC News
+
+We've hit 30 team members! Our team is distributed all over the world and has
+grown so much that we now have two all-hands meetings! Affectionately called
+UTC + and UTC -, these meetings honor all our different time zones while
+allowing the other group to watch via recording when they are awake! You know
+we're all about solving complicated problems. 💪🏼
+
+![](/uploads/images/2021-05-15/team-map.png)_Our team_
+
+## New Team Members
+
+[Svetlana Sachkovskaya](https://www.linkedin.com/in/svetlana-sachkovskaya/) is
+originally from Belarus, but is currently living in Poland. She has been a full
+stack developer for over seven years. She loves traveling, meeting new people
+and is excited to work on open source software. In her spare time you may find
+her dancing the tango! 💃🏻 Welcome Sveta!
+
+Exemplifying our diverse team in one fell swoop,
+[Casper da Costa-Luis](https://www.cdcl.ml) has lived in three continents. He
+has been working on DVC for a couple of years and is a long-standing contributor
+to open source. He now joins us on the CML & Docs teams after completing his PhD
+in Medical Imaging. Fun facts about Casper include his becoming the U18 chess
+champion of Kenya when he was 14 and being a qualified SCUBA diver. 🤿
+
+[Emre Sahin](https://github.com/iesahin) joins us on the DevRel team as a
+techinical writer/ML enthusiast/AI dreamer/tutorial builder from Instanbul,
+Turkey. A self-described zelaot for technologies, Emre has worked in many
```

Comment:
```suggestion
Turkey. A self-described zealot for technologies, Emre has worked in many
```

Reply:
Thank you for fixing my things :)) @casperdcl @skshetry